### DISCUSSION TITLE
'Tooltips' for definitions and short explanations when mousing over content

### DISCUSSION THREAD

**Discussion by michael-rowe** [outline/outline#5380] [Ideas]
Hi everyone. I'm not sure if 'tooltips' is the correct word, but I'd love to be able to have definitions and short explanations available as a short 'tooltip' that shows up in a notification popup when you mouseover defined text.

**Comment by routerino** [comment]
Yep, it would be nice to be able to add an inline info icon that has a tooltip popup. This would be especially useful for templates where you want to add a lot of supporting text to placeholders without physically creating a block the people have to remove.
